airflow/operators/http_operator.py

```diff
@@ -0,0 +1,33 @@
+import logging
+
+from airflow.hooks import HttpHook
+from airflow.models import BaseOperator
+from airflow.utils import apply_defaults
+from airflow.utils import AirflowException
+
+class SimpleHttpOperator(BaseOperator):
+    """
+    Calls a url 
+    """
+
+    template_fields = ('url',)
+    template_ext = ()
+    ui_color = '#f4a460'
+
+    @apply_defaults
+    def __init__(self, url, method='POST', data=None, headers={}, http_conn_id='http_default', *args, **kwargs):
+        super(SimpleHttpOperator, self).__init__(*args, **kwargs)
+        self.http_conn_id = http_conn_id
+        self.method = method
+        self.url = url
+        self.data = data
+        self.headers = headers
+
+    def execute(self, context):
```

Comment:
As you mentioned it could be nice to accept an optional regex to try to match against the response, or even better: a function (or lambda) that expects to receive a response object and returns a bool. Then you `raise` if the calls evals to False.